src/main/scala/li/cil/oc/util/InventoryUtils.scala

```diff
@@ -120,32 +137,31 @@ object InventoryUtils {
    * also be achieved by a check in the consumer, but it saves some unnecessary
    * code repetition this way.
    */
-  def extractFromInventorySlot(consumer: (ItemStack) => Unit, inventory: IInventory, side: EnumFacing, slot: Int, limit: Int = 64) = {
+  def extractFromInventorySlot(consumer: (ItemStack) => Unit, inventory: IItemHandler, slot: Int, limit: Int = 64): Boolean = {
     val stack = inventory.getStackInSlot(slot)
     (stack != null && limit > 0) && {
```

Comment:
Throw in that `&& stack.stackSize > 0` here, too, seeing as that check existed here before, too. At least it'll save us some unnecessary work.